lib/parse-message.js

```diff
@@ -0,0 +1,47 @@
+'use strict';
+module.exports = exports = {};
+
+
+exports.nickCommand = function(message, socket) {
+  console.log('username: ' + socket.username);
+  socket.username = message.split(' ').slice(1).join(' ').trim();
+  socket.write(`you are now ${socket.username}\n`);
+
+};
+
+exports.dmCommand = function(message, array){
+ // "/dm user_30 hello how are you"
```

Comment:
Same I said with console.logs, do the same with comments, as in be aware of getting rid of these before you submit code. 

If the comments are more for outsiders to understand a bit what your code does - i.e., maybe for this particular function, you could write at the top, "command to send direct message to another user", that that would be fine. This comment doesn't really tell me anything if I knew nothing else about your project.